css/civicrmNavigation.css

```diff
@@ -91,6 +91,7 @@ li.menu-separator.active{
   display:block;
   font-weight:normal;
   text-decoration:none;
+  border-bottom-style:none;
```

Comment:
if this means both civicrm and drupal8 use li.menu-separator, i might also add a `.page-civicrm` (or d8 equivalent) prefix to this override, to avoid clobbering the underlying theme